libs/langchain/langchain/vectorstores/deeplake.py

```diff
@@ -766,14 +765,8 @@ def from_texts(
             ValueError: If 'embedding' is provided in kwargs. This is deprecated,
                 please use `embedding_function` instead.
         """
-        if embedding:
-            raise ValueError(
-                "using embedding as embedidng_functions is deprecated. "
-                "Please use `embedding_function` instead."
-            )
-
         deeplake_dataset = cls(
-            dataset_path=dataset_path, embedding_function=embedding_function, **kwargs
+            dataset_path=dataset_path, embedding_function=embedding, **kwargs
```

Comment:
should embedding_function just be renamed to embedding as well?

(you may want to keep embedding_function around on init for backwards compatibility, but you can reroute to self.embedding if it's passed in a log a warning)